.travis.yml

```diff
@@ -151,7 +151,7 @@ env:
     PARENT_GROUP=all
     HOST_GROUP=db_server
     PLAYBOOK_LOCATION=/aurora/deploy/deploy_db_server.yml
-    EXTRA_VAR_DEF='{"deploy_iptables": false, "install_ppas": false, "skip_glusterfs": true}'
+    EXTRA_VAR_DEF='{"deploy_iptables": false, "install_postgres": false, "skip_glusterfs": true}'
```

Comment:
Expanding on this, we may want to just remove the `install_postgres` variable completely from here (since the default is to install open-source Postgres, it should now work on Travis), and add tags to the various tasks instead.